Nodejs/Tests/AnalysisTests/AnalysisFile.cs

```diff
@@ -131,6 +131,20 @@ static class Analysis {
             List<AnalysisFile> files = new List<AnalysisFile>();
             foreach (var file in Directory.GetFiles(directory, "*", SearchOption.AllDirectories)) {
```

Comment:
I try to find place where this could be placed, but did not find any public helper class which accessible from AnalysisDryver.

@DinoV could you show me 
1. in which public helper could I place this code 
2. or alternatively I could add InternalsVisibleTo("AnalysisDriver") to Analyisys assembly.